sark/code/line.py

```diff
@@ -185,6 +185,11 @@ def xrefs_from(self):
         return imap(Xref, idautils.XrefsFrom(self.ea))
 
     @property
+    def has_xrefs_from(self):
+        """Are there Xrefs from this line."""
+        return len(list(self.xrefs_from)) != 0
```

Comment:
This is quite wasteful, as it consumes the entire generator. A better alternative will be:

```
for _ in self..xrefs_from:
    return True
return False
```